server/seeds/demoSeed.js

```diff
@@ -0,0 +1,31 @@
+const User = require("../models/User");
+const mongoose = require("mongoose");
+require('dotenv').config({ path: '../.env' })
+
+const users = [
+  new User({
+    name: process.env.DEMO_USER_NAME,
+    email: process.env.DEMO_USER_EMAIL,
+    password: process.env.DEMO_USER_PASSWORD,
+  })]
+
+mongoose
+  .connect(process.env.MONGO_URI)
+  .catch(err => {
+    console.log('error while connecting to mongoose: ', err.stack);
+    process.exit(1);
+  })
+  .then(() => {
+    console.log("db connection ready");
+  });
```

Comment:
Not the most important thing, but just to point out: Optionally, you can also import and use the `connectDB` function from [`db.js`](https://github.com/hatchways/team-cedar/blob/add/demo-user-BE/server/db.js) here. You would still need to handle any errors yourself though 👍🏼